### ISSUE TITLE
eslintIntegration doesn't work when .eslintrc depends on global packages

### ISSUE THREAD

**Issue by infernalmaster** [prettier/prettier-vscode#397]
So I have this .eslintrc 

```
{
  "parser": "babel-eslint",
  "extends": [
    "standard",
    "standard-react"
  ],
  "env": {
    "browser": true,
    "node": true
  },
  "rules": {}
}
```

My npm packages:
```
yarn add --dev eslint babel-eslint eslint-config-standard eslint-config-standard-react eslint-plugin-react eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node eslint-plugin-react prettier prettier-eslint-cli
```

With this setup I can format files with CLI
```
prettier-eslint --write my.file
```

But VSCode plugin ignores .eslintrc and starts to works properly only when all that packages are installed locally

**Comment by gregbty** [comment]
I am seeing this issue too on Windows only, but all my packages are local. The same project didn't have this issue a couple of weeks ago, so I'm assuming the plugin updated.

**Comment by ntotten** [comment]
This should be resolved in 3.13+. We now support global modules. Please reopen if you still see problems.
